container/go/cmd/digester/digester.go

```diff
@@ -16,23 +16,28 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"flag"
 	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/bazelbuild/rules_docker/container/go/pkg/compat"
 	"github.com/bazelbuild/rules_docker/container/go/pkg/utils"
 	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/types"
 	"github.com/pkg/errors"
 )
 
 var (
-	dst    = flag.String("dst", "", "The destination location of the digest file to write to.")
-	src    = flag.String("src", "", "Path to the config.json when -format is legacy, path to the index.json when -format is oci or path to the image .tar file when -format is docker.")
-	format = flag.String("format", "", "The format of the source image, (oci, legacy, or docker). The docker format should be a tarball of the image as generated by docker save.")
-	layers utils.ArrayStringFlags
+	dst     = flag.String("dst", "", "The destination location of the digest file to write to.")
+	src     = flag.String("src", "", "Path to the config.json when -format is legacy, path to the index.json when -format is oci or path to the image .tar file when -format is docker.")
+	format  = flag.String("format", "", "The format of the source image, (oci, legacy, or docker). The docker format should be a tarball of the image as generated by docker save.")
+	tarball = flag.String("tarball", "", "The path to a legacy base image in tarball form, optional.")
```

Comment:
Put `Optional:` in the beginning of the help text. Also mention this is only relevant when format is legacy. Probably name the flag `legacyBaseImage` to make this more explicit in the flag name.